.reuse/dep5

```diff
@@ -0,0 +1,29 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cloud-metricswatcher
+Upstream-Contact: Yavor Mladenov <yavor.mladenov@sap.com>
+Source: https://github.com/SAP/cloud-metrics-dashboard
```

Comment:
Sorry I just saw that the link is also pointing to another repository :)

Reply:
:D Fixed